roles/bwc_smoketests/templates/rbac_assignments/assignments.yml.j2

```diff
@@ -0,0 +1,5 @@
+---
+
+  username: {{ item.name }}
+  roles:
```

Comment:
and 0 spaces here.

Otherwise `{{ item.roles | to_nice_yaml(2) | indent(2) }}` won't work correctly if we add some complex data there